src/ui/MessageSearchItem/getCreatedAt.ts

```diff
@@ -14,20 +15,20 @@ interface GetCreatedAtProps {
 export default function ({
   createdAt,
   locale,
-  stringSet,
+  stringSet = getStringSet('en'),
```

Comment:
@HoonBaek  Any reason we have this initialization here? I have a concern that this might cause unintended layout shift.

Reply:
It's for setting default value, I think it won't ruin the layout

Reply:
Actually same comment with https://github.com/sendbird/sendbird-uikit-react/pull/1221/files#r1774659453